kasa/httpclient.py

```diff
@@ -103,6 +118,9 @@ async def post(
                 f"Unable to query the device: {self._config.host}: {ex}", ex
             ) from ex
 
+        if self._wait_between_requests:
+            self._last_request_time = time.time()
```

Comment:
```suggestion
        self._last_request_time = time.time()
```
I'm not sure if it's worth to guard this, just to keep it simple?

Reply:
I figured there is more of a performance concern calling `time.time()` for every device regardless of whether they need it. It's only one device we know of that has this requirement.

Reply:
Yeah, fair enough. Maybe add a quick comment there why this is done, to avoid any guess work in case this gets refactored somehow later on.

Reply:
Done